.circleci/config.yml

```diff
@@ -245,6 +245,29 @@ jobs:
               -t owasp/zap2docker-weekly zap-baseline.py \
               -t http://server:8080 \
               -c zap.conf -I -i
+  accessibility_scan:
+    executor: docker-executor
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: Run migrations ci
+          command: yarn run db:migrate:ci
+      - run:
+          name: Build
+          command: yarn build
+      - run:
+          name: Start server
+          command: BYPASS_AUTH=true yarn start:ci
+          background: true
+      - run:
+          name: Run axe
+          command: |
+            mkdir reports
+            sleep 5
```

Comment:
Why is the sleep necessary? To let the server have time to start up?

Reply:
This one hurt a little. Unfortunately, yes, the server needs time to start up. The axe command fails without it.